components/logins/src/logins.udl

```diff
@@ -140,12 +143,18 @@ interface LoginStore {
     [Throws=LoginsApiError]
     boolean delete([ByRef] string id);
 
+    [Throws=LoginsApiError]
+    boolean delete_local([ByRef] string id);
```

Comment:
I think we should come up with a better name, because I can imagine someone accidentally calling this rather than `delete`. I'm not too sure what a good name is though, but I think a long and ugly name is fine given it's just for this special edge-case - eg, something like `delete_local_for_remote_replacement()` or something? And maybe a comment to explain why it shouldn't actually be used in normal cases.